roles/rsyslog/tasks/main.yml

```diff
@@ -1,3 +1,11 @@
+- name: Set version specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
+    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
```

Comment:
you should have a `default.yml` here to catch the case where the `ansible_os_family` and `ansible_distribution` is unknown.